pkg/downloader/manager.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/pkg/errors"
 	"sigs.k8s.io/yaml"
 
+	"helm.sh/helm/v3/internal/dep/fs"
```

Comment:
This should be 
```suggestion
	"helm.sh/helm/v3/internal/third_party/dep/fs"
```

Reply:
Change you asked for is done. Sorry this was very lame mistake on my part.